src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
The "Unresolved Variable"-IDE-problem is absolutely no argument to me. Please don't put missing IDE features (or however you call it) above application architecture.

The problems having this in a "SDK" are multiple nature.
1. A SDK isn't provided out-of-the-box, it needs to be integrated by the developer manually.
2. The SDK replicates all the identically Helper code in every plugin.
3. It's another repository that needs to be maintained. In fact the plugin interface is part of the UI.

What do you mean with: "Injecting everything into the plugin window feels wrong." ?

And in fact, basically we only expose the PluginDispatcher.
The PluginHelper functions are only little helpers that are calling the PluginDispatcher.
The other files are constants-files.
Nothing special, everything at one place.